com.ibm.wala.cast.java/src/testFixtures/java/com/ibm/wala/cast/java/test/JavaIRTests.java

```diff
@@ -721,6 +727,38 @@ public void testMiniaturSliceBug() throws IllegalArgumentException, CancelExcept
     Assert.assertEquals(2, SlicerUtil.countPutfields(slice));
   }
 
+  @Test
+  public void testThinSlice() throws CancelException, IOException {
+    List<String> sources =
+        Collections.singletonList(getTestSrcPath() + File.separator + "MiniaturSliceBug.java");
+    Pair<CallGraph, PointerAnalysis<? extends InstanceKey>> x =
+        runTest(sources, rtJar, simpleTestEntryPoint(), emptyList, true, null);
+
+    @SuppressWarnings("unchecked")
```

Comment:
@liblit somehow ECJ doesn't pay attention to this suppression and reports an unchecked cast error on the next line anyway, which is why I changed the ECJ config.  I also don't know how to write this code to avoid an unchecked cast, without potentially changing a lot of other code (and it didn't seem worth the effort).  Help welcome on making the suppression work :-)  Maybe it's because it's in text fixture code?